#377:Added a package reference to DeathwatchActor

girish17 opened the pull request:
It might be useful to refer the DeathwatchActor resource while reading the package documentation for Location service.

girish17 committed:
#377:Added a package reference to DeathwatchActor

skvithalani commented:
Hello @girish17 ,

Thanks for the change. In order for me to merge it, please solve the scalastyle check that failed.

girish17 committed:
Corrected for scalastyle check

Issue: File line length should not exceed 160 chars.
Lines that are too long can be hard to read and horizontal scrolling is annoying.
